src/closure/createClosure.ts

```diff
@@ -1038,18 +1088,34 @@ async function getOrCreateEntryAsync(
     if (normalizedModuleName) {
       await captureModuleAsync(normalizedModuleName);
     } else if (obj instanceof Function) {
-      // Serialize functions recursively, and store them in a closure property.
-      entry.function = await analyzeFunctionInfoAsync(
-        obj,
-        context,
-        serialize,
-        logInfo
-      );
+      // if not invoked, let the property be treated like an object
```

Comment:
meaning attempt to only serialize used properties?

Reply:
i.e. tree-shake?

Reply:
right, previous behavior was that anything recognized as a function `instanceof Function` would be fully serialized, meaning static properties made a mess.

Reply:
And namespaces more so :-)